src/qt/buynamespage.cpp

```diff
@@ -78,6 +101,42 @@ void BuyNamesPage::onRegisterNameAction()
     ui->registerNameButton->setDefault(true);
 }
 
+// Returns empty string if available, otherwise a description of why it is not
+// available.
+QString BuyNamesPage::name_available(const QString &name) const
+{
+    std::string strName = name.toStdString();
+    LogPrintf ("wallet attempting name_show: name=%s\n", strName);
```

Comment:
I think we should avoid `LogPrintf`, and instead use the `LogPrint` method with a log category.

Reply:
There are other places in the name GUI code that use `LogPrintf`; is it okay if I fix all of them in this PR?

Reply:
Yes definitely.

Reply:
I take it `BCLog::QT` is the right category for anything in the `qt` subdirectory?

Reply:
Yes that sounds reasonable to me.